site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Installing the Agones on Azure Kubernetes Service"
+linkTitle: "Azure"
+weight: 20
+description: >
+  You can deploy Kubernetes cluster on Azure Kubernetes Service and install Agones using terraform.
+---
+
+Install `az` utility by following [these instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
```

Comment:
There is a heading (##) for uninstall but not for install. Just diving into the instructions doesn't match the style elsewhere on the site.